curated_transformers/tests/models/electra/test_encoder.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from curated_transformers.models.electra.encoder import ELECTRAEncoder
+
+from ...compat import has_hf_transformers
+from ...conftest import TORCH_DEVICES
+from ..util import (
+    assert_encoder_output_equals_hf,
+)
+
+
+@pytest.mark.skipif(not has_hf_transformers, reason="requires huggingface transformers")
+@pytest.mark.parametrize("torch_device", TORCH_DEVICES)
+@pytest.mark.parametrize("with_torch_sdp", [False, True])
+@pytest.mark.parametrize(
+    "model_name",
+    [
+        "jonfd/electra-small-nordic",
+        "Maltehb/aelaectra-danish-electra-small-cased",
+        "google/electra-small-discriminator",
+    ],
```

Comment:
I checked using a variety of models, but I imagine you might want to replace this with a dummy Electra model.